packages/dx-react-grid/src/plugins/data-type-provider.test.tsx

```diff
@@ -1,14 +1,35 @@
 import * as React from 'react';
 import { mount } from 'enzyme';
-import { getAvailableFilterOperationsGetter } from '@devexpress/dx-grid-core';
-import { PluginHost } from '@devexpress/dx-react-core';
+import {
+  getAvailableFilterOperationsGetter,
+  TABLE_DATA_TYPE,
+} from '@devexpress/dx-grid-core';
+import {
+  PluginHost,
+  Template,
+  TemplateConnector,
+  TemplatePlaceholder,
+} from '@devexpress/dx-react-core';
 import { pluginDepsToComponents, getComputedState, setupConsole } from '@devexpress/dx-testing';
 import { DataTypeProvider } from './data-type-provider';
 
 jest.mock('@devexpress/dx-grid-core', () => ({
   getAvailableFilterOperationsGetter: jest.fn(),
 }));
 
+const defaultDeps = {
+  getter: {
+    tableColumns: [
+      { key: `${TABLE_DATA_TYPE}_a`, type: TABLE_DATA_TYPE, column: { name: 'a' } },
```

Comment:
Just `key: 'a'` is enough

Reply:
We don't need `type` here. Besides that, `TABLE_DATA_TYPE` is undefined since grid-core is mocked